### PR TITLE
A few quality of life changes

### PR DESCRIPTION
**Pull request by aidansteele**
Sorry for bundling up three unrelated changes into a single PR - I can split them out into three PRs if that is preferable. I didn't know which would be less annoying 😄

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,22 +35,22 @@ Options:
...
+By default, amp-eks-iam creates the role and the privileges for the
 "prometheus" Kubernetes namespace and service account. You can specify your own
 namespaces and service accounts. For example, if you are deploying Grafana Agent
 as explained [in this article](https://aws.amazon.com/blogs/opensource/configuring-grafana-cloud-agent-for-amazon-managed-service-for-prometheus/),
 use the following command:
 
 ```
 $ amp-eks-iam \
-   -account=999999999999 -region=us-east-1 -cluster=eks-cluster \
+   -region=us-east-1 -cluster=eks-cluster \
```
Comment (rakyll) [inline README.md:46]: We should remove -account from help.go as well.

**Review by rakyll — APPROVED** [review]
Thank you so much, left a comment. Otherwise, looks good.

File: main.go [hunk 2]
```diff
@@ -20,6 +20,8 @@ import (
 	"errors"
 	"flag"
 	"fmt"
+	"github.com/aws/aws-sdk-go/aws/credentials/stscreds"
```
Comment (rakyll) [inline main.go]: One nit, split standard library imports from third party imports with a line.

```
import (
    "errors"
     "flag"
      ....

      "github.com/aws/aws-sdk-go/aws/credentials/stscreds"
      "github.com/aws/aws-sdk-go/service/sts"
)
```

**Comment by aidansteele** [comment]
Oh great points, I've pushed some additional changes to address that.

**Comment by rakyll** [comment]
Thank you!

**Pull request merged**